src/main/scala/org/phenoscape/kb/PresenceAbsenceOfStructure.scala

```diff
@@ -122,7 +122,7 @@ object PresenceAbsenceOfStructure {
         dataset.setStateForTaxon(matrixTaxon, character, stateToAssign)
       }
       val date = new SimpleDateFormat("y-M-d").format(Calendar.getInstance.getTime)
-      dataset.setPublicationNotes(s"Generated from the Phenoscape Knowledgebase on $date by Ontotrace query:\n* taxa: ${taxonClass.asOMN.getLiteralLexicalForm}\n* entities: ${entityClass.asOMN.getLiteralLexicalForm}")
+      dataset.setPublicationNotes(s"Generated from the Phenoscape Knowledgebase on $date by Ontotrace query:\n* taxa: ${taxonClass.asOMN.getLiteralLexicalForm}\n* entities: ${entityClass.asOMN.getLiteralLexicalForm}\n* variableOnly: ${variableOnly}")
```

Comment:
This is user-facing text, so instead of camelcase, just write "variable only". Sorry this is kind of nit-picking. 🤓

Reply:
Actually I don't think it's a bad idea to have a full listing of parameters and their values, as opposed to having to infer from the absence of _variable only_ that this parameter was set to false. (This inference would also have to rely on the assumption that if variable only had been set to true, it would have been reported. Obviously, this wasn't the case in the past.)

Reply:
Oh, sorry, that wasn't quite what I meant. I meant it should be `* variable only: true` rather than `variableOnly: true`.

Reply:
Nitpicking is good. I like nitpicking. Leads to perfection. Soon you'll get fewer opportunities to nitpick on me :p

Reply:
Okay, then excellent! Here is just a little more: you have added a space, but "Only" is still capitalized. :-)

Reply:
Haha.. Keep it coming B-)